neutron-org/neutron-airdrop-transfer
# feat: airdrop transfer contract #ntrn-116 (recreated pr)

https://hadronlabs.atlassian.net/browse/NTRN-116

How to run integration tests:
1. Checkout gaia `v13.0.0` for building image, checkout neutron `main` branch.
2. in root: `make build`
3. `cd integration-tests`
4. `make build-all`
5. run `yarn`
6. run `yarn test`
7. tests should pass!
